1_turb_abl_fsi/template_files/frontier_aprepro.txt

```diff
@@ -0,0 +1,20 @@
+# {NALU_MESH = "/lustre/orion/cfd116/proj-shared/ashesh/tower+rotor_correct_height/tower_and_blades.exo"}
```

Comment:
This line has orion/cfd116/proj-shared, and the next has orion/proj-shared/cfd116.  Is that right?

Reply:
I am not intimately familiar with the file system, but the order of <project name> and  proj-shared or scratch has seemed interchangeable on lustre and previously on summit too.

Reply:
These are the files that I ran with. I'm not sure of the details, but I will keep them as is for this PR. We can tweak in a follow on PR if need be.